testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -1372,6 +1372,7 @@ private void testDataMapping(DataMappingTestSetup dataMappingTestSetup)
                 .add(new DataMappingTestSetup("double", "DOUBLE '1234567890123.123'", "DOUBLE '9999999999999.999'"))
                 .add(new DataMappingTestSetup("decimal(5,3)", "12.345", "99.999"))
                 .add(new DataMappingTestSetup("decimal(15,3)", "123456789012.345", "999999999999.99"))
+                .add(new DataMappingTestSetup("date", "DATE '1582-10-05'", "DATE '1582-10-14'")) // julian->gregorian switch
```

Comment:
Add a test case before the switch like 1582-10-04 
however, date `0001-01-01`  is more expected in real life -- people rarely store real historical dates in databases. OTOH users been witnessed using special values like  `0001-01-01` intentionally.
So,  `0001-01-01` is worth being tested as practical value.
and `1582-10-04 ` is worth being tested as the value where Julian/Gregorian vs proleptic Gregorian calendar difference is maximal.

please add both